aws_jobs/cyhy-data-extract.py

```diff
@@ -328,63 +334,118 @@ def main():
     else:
         orgs = []
 
+    default_projection = {"key": False}
+
     cyhy_collection = {
         "host_scans": {
-            "owner": {"$in": orgs},
-            "time": {"$gte": start_of_data_collection, "$lt": end_of_data_collection},
+            "query": {
+                "owner": {"$in": orgs},
+                "time": {
+                    "$gte": start_of_data_collection,
+                    "$lt": end_of_data_collection,
+                },
+            },
+            "projection": default_projection,
         },
-        "port_scans": {
-            "owner": {"$in": orgs},
-            "time": {"$gte": start_of_data_collection, "$lt": end_of_data_collection},
+        "hosts": {
+            "query": {
+                "owner": {"$in": orgs},
+                "last_change": {
+                    "$gte": start_of_data_collection,
+                    "$lt": end_of_data_collection,
+                },
+            },
+            "projection": default_projection,
         },
-        "vuln_scans": {
-            "owner": {"$in": orgs},
-            "time": {"$gte": start_of_data_collection, "$lt": end_of_data_collection},
+        "port_scans": {
+            "query": {
+                "owner": {"$in": orgs},
+                "time": {
+                    "$gte": start_of_data_collection,
+                    "$lt": end_of_data_collection,
+                },
+            },
+            "projection": default_projection,
         },
-        "hosts": {
-            "owner": {"$in": orgs},
-            "last_change": {
-                "$gte": start_of_data_collection,
-                "$lt": end_of_data_collection,
+        "requests": {
```

Comment:
@mcdonnnj - I was thinking about it this morning and I think we should add a brief comment here explaining why every document in the `requests` collection is being exported, rather than a subset based on a date field.

Reply:
Good thought, I added an informative comment in 401fae5